### PR TITLE
[GECO-44] find image file with and without preceding slash

### PR DESCRIPTION
**Pull request by snilapwa**

### CODE REVIEW

File: giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/aspects/access/RestSecurityAspect.java [hunk 1]
```diff
@@ -262,12 +263,24 @@ public Object checkImageAccess(ProceedingJoinPoint joinPoint, ImageAccessCheck c
...
-                
+
                 IFile file = filesManager.getFileByPath(imagePath);
+
+                if (file == null) {
+                    if (imagePath.startsWith(File.separator)) {
+                        imagePath = imagePath.substring(1);
+                    } else {
+                        imagePath = File.separator + imagePath;
+                    }
```
Comment (jdamerow) [inline giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/aspects/access/RestSecurityAspect.java]: make that a ternary operator

File: giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/rest/DigilibPassthroughController.java [hunk 2]
```diff
@@ -135,11 +135,17 @@
...
-        }
-        
         IFile file = filesManager.getFileByPath(fn);
+
+        if (file == null) {
+            if (fn.startsWith(File.separator)) {
+                fn = fn.substring(1);
+            } else {
+                fn = File.separator + fn;
+            }
```
Comment (jdamerow) [inline giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/rest/DigilibPassthroughController.java]: same here

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/aspects/access/RestSecurityAspect.java (hunk 1 above)
Comment (snilapwa) [reply giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/aspects/access/RestSecurityAspect.java]: changed

File: giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/rest/DigilibPassthroughController.java (hunk 2 above)
Comment (snilapwa) [reply giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/rest/DigilibPassthroughController.java]: changed

**Comment by jdamerow** [comment]
ok to test

**Review by jdamerow — APPROVED** [review]

**Pull request merged**